src/main/java/uk/gov/hmcts/divorce/common/model/Applicant.java

```diff
@@ -106,11 +107,66 @@
     @JsonUnwrapped(prefix = "Solicitor")
     private Solicitor solicitor;
 
+
+    @CCD(
+        label = "Has the respondent read the application for divorce?",
```

Comment:
worth checking should we be referring respondent or applicant2 ?

Reply:
Confirmation - it should be applicant 1 & respondent